pylint/checkers/base/comparison_checker.py

```diff
@@ -58,6 +59,13 @@ class ComparisonChecker(_BasicChecker):
             "comparison-with-itself",
             "Used when something is compared against itself.",
         ),
+        "R0125": (
+            "Comparison between constants: '%s %s %s' is also a constant, remove the comparison and consider a refactor",
```

Comment:
```suggestion
            "Comparison between constants: '%s %s %s' has a constant value",
```

The `R` in `R0125` implies that the code should be refactored so I think we can leave that out of the message here.